python-sdk/pachyderm_sdk/api/storage/extension.py

```diff
@@ -0,0 +1,166 @@
+"""Handwritten classes/methods that augment the existing Storage API."""
+
+import os
+from pathlib import Path
+from typing import Union
+
+from betterproto import which_one_of
+
+from ..cdr.resolver import (
+    CdrResolver,
+    Ref,
+    Cipher,
+    Compress,
+    Concat,
+    ContentHash,
+    Http,
+    SizeLimits,
+    Slice,
+)
+from . import FilesetStub as _GeneratedApiStub
+from . import FileFilter, PathRange
+from pachyderm_sdk.constants import CDR_CACHE_LOCATION
+
+
+class ApiStub(_GeneratedApiStub):
+
+    def assemble_fileset(
+        self,
+        fileset: str,
+        path: Union[str, PathRange],
+        destination: os.PathLike,
+        *,
+        cache_location: os.PathLike = CDR_CACHE_LOCATION,
+        fetch_missing_chunks: bool = True,
+        http_host_replacement: str = "",
+    ) -> None:
+        """Assemble the data contained at the specified path within the fileset
+        and write that data to the specified destination directory. This method
+        uses a local cache and will automatically fetch missing data from PFS,
+        unless disabled.
+
+        Parameters
+        ----------
+        fileset : str
+            The UUID of the fileset.
+        path : str
+            The path, regex, or PathRange within the fileset to assemble.
+            A value of "/" assembles the entire fileset.
+        destination : os.PathLike
+            The local directory at which to assemble the fileset.
+        cache_location : os.PathLike (kwarg only)
+            The location of the chunk cache. This is also configurable thru ENV VAR.
+        fetch_missing_chunks : bool (kwarg only)
+            If true, fetch any missing chunks needed to assemble the fileset.
+            If false, raise an error if any chunks are missing.
+        http_host_replacement : str
```

Comment:
Should this also include the `(kwarg only)` note like the prior paramaters? Same comment for `fetch_chunks`.